app/models/claim.rb

```diff
@@ -50,6 +49,9 @@ class Claim < ActiveRecord::Base
   before_update -> { secondary_claimants.destroy_all },
     if: :additional_claimants_csv_changed?
 
+  before_update :remove_additional_claimants_csv!,
+    if: -> (c) { c.secondary_claimants.any? }
```

Comment:
It looks like the arity is checked. I remember getting `ArgumentError`s earlier in the project doing this. Gonna change it to this 

```ruby
before_update :remove_additional_claimants_csv!,
  if: -> { secondary_claimants.any? }
```